etc/adminhtml/system/adyen_split_payment.xml

```diff
@@ -29,7 +29,7 @@
         <comment>
             <![CDATA[
                 <p>
-                It is possible to do split payments with GiftCards please configure here what refund strategie you want to use
+                It is possible to do split payments with GiftCards please configure here what refund strategy you want to use.
```

Comment:
Rephrasing it a bit. And Split Payment is now related to AFP, the right label would be Partial Payment
```suggestion
                It is possible to do partial payments with GiftCards. Set here which refund strategy to use.
```

Reply:
Should the Label on line 27 and/or the filename be renamed as well?

Reply:
Makes sense yes. Are you able to change the label and filename @peterojo ?

Reply:
I can but there's quite a lot of references to split payments in the codebase, and we would also need to test that it still works.
![Screen Shot 2021-09-09 at 3 46 37 PM](https://user-images.githubusercontent.com/5240725/132697573-1e805cff-ad2c-4e01-94f0-e1ba033dec1c.png)

I would rather make it a separate effort. Here we just wanted to correct a typo 😄

Reply:
I've created a ticket [PW-5431] to update all references to split payments